pkg/kubectl/cmd/resource/get.go

```diff
@@ -348,6 +348,10 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 				}
 				continue
 			}
+			// Don't need wrap sortingPrinter in printing each object
```

Comment:
I think this is because we have sorting logic in both `get` command and `printer`. But for those non-generic printer, we have to sort objects in `get` logic.